src/pages/onboarding/create.vue

```diff
@@ -262,7 +263,14 @@ export default {
       }
     },
 
-    async onNextStep () {
+    scrollToNextStep (nextStep) {
+      if (!this.pastSteps.includes(nextStep)) {
+        this.pastSteps.push(nextStep)
+      }
+      setTimeout(() => { document.getElementById(nextStep).scrollIntoView({ behavior: 'smooth', block: 'center' }) }, 400)
```

Comment:
@evgenibir I think its better to use[ build-in quasar function for this](https://quasar.dev/quasar-utils/scrolling-utils)

example"
```
import { scroll } from 'quasar'
const { getScrollTarget, setVerticalScrollPosition } = scroll
```

Reply:
@arsenijesavic Unfortunately, scroll quasar does not contain some functions, for example, benaivor, also, in the application, the scroll is implemented in a separate component (not across the entire browser window)